src/util/debug_module.js

```diff
@@ -508,6 +508,7 @@ function log_syslog_builder(syslevel) {
  * log_builder will format the message with prefix. 
 */
 function log_event(event) {
+    if (config.DB_TYPE !== 'none') return; // currently event logging is only in NC NSFS
```

Comment:
Event logging option to other deployment is not a bad decision, It will just print one more log in log file and it will help debugging at the end.
@romayalon @shirady WDYT?

Reply:
@naveenpaul1 currently the logs are not accurate for containerized env, if we want them we need a bit change them (for example "host" field), and currently some of the logs are on S3 errors and we have those printed in the log, having it twice might fill the logs with those printings (for example `OBJECT_GET_FAILED` on a client that crawls many files name and tries to GET them).